.eslintcache

```diff
@@ -0,0 +1 @@
+[{"/Users/sgguruda/work/git_codebase/osd-3.0/OpenSearch-Dashboards/plugins/dashboards-observability/public/components/custom_panels/custom_panel_table.tsx":"1","/Users/sgguruda/work/git_codebase/osd-3.0/OpenSearch-Dashboards/plugins/dashboards-observability/public/components/custom_panels/custom_panel_view.tsx":"2","/Users/sgguruda/work/git_codebase/osd-3.0/OpenSearch-Dashboards/plugins/dashboards-observability/public/components/custom_panels/helpers/add_visualization_popover.tsx":"3","/Users/sgguruda/work/git_codebase/osd-3.0/OpenSearch-Dashboards/plugins/dashboards-observability/public/components/custom_panels/panel_modules/visualization_container/visualization_container.tsx":"4","/Users/sgguruda/work/git_codebase/osd-3.0/OpenSearch-Dashboards/plugins/dashboards-observability/public/components/custom_panels/panel_modules/visualization_flyout/visualization_flyout.tsx":"5"},{"size":12806,"mtime":1675271951921,"results":"6","hashOfConfig":"7"},{"size":20913,"mtime":1675356693156,"results":"8","hashOfConfig":"7"},{"size":2112,"mtime":1675271951922,"results":"9","hashOfConfig":"7"},{"size":8986,"mtime":1675312992200},{"size":11638,"mtime":1675271951923,"results":"10","hashOfConfig":"7"},{"filePath":"11","messages":"12","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},"1i553ik",{"filePath":"13","messages":"14","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},{"filePath":"15","messages":"16","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},{"filePath":"17","messages":"18","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},"/Users/sgguruda/work/git_codebase/osd-3.0/OpenSearch-Dashboards/plugins/dashboards-observability/public/components/custom_panels/custom_panel_table.tsx",[],"/Users/sgguruda/work/git_codebase/osd-3.0/OpenSearch-Dashboards/plugins/dashboards-observability/public/components/custom_panels/custom_panel_view.tsx",[],"/Users/sgguruda/work/git_codebase/osd-3.0/OpenSearch-Dashboards/plugins/dashboards-observability/public/components/custom_panels/helpers/add_visualization_popover.tsx",[],"/Users/sgguruda/work/git_codebase/osd-3.0/OpenSearch-Dashboards/plugins/dashboards-observability/public/components/custom_panels/panel_modules/visualization_flyout/visualization_flyout.tsx",[]]
```

Comment:
Should we remove this?

Reply:
I think you're looking at an older commit. It is already removed in the next one https://github.com/opensearch-project/dashboards-observability/pull/244/commits/c584a93f0748c2ff6f0d2783fa102e9ae27601e7